docs/data/data-grid/editing/editing.md

```diff
@@ -456,109 +456,7 @@ Instead, use the buttons available in each row or in the toolbar.
-The code above is already enough to display different options in the **Account** column based on the value selected in the **Type** column.
-The only task left is to reset the account once the type is changed.
-This is needed because the previously selected account will not exist now in the options.
-To solve that, you can create a custom edit component, reusing the built-in one, and pass a function to the `onValueChange` prop.
-This function should call `apiRef.current.setEditCellValue` to reset the value of the other field.
-
-```tsx
-const CustomTypeEditComponent = (props: GridEditSingleSelectCellProps) => {
-  const apiRef = useGridApiContext();
-
-  const handleValueChange = async () => {
-    await apiRef.current.setEditCellValue({
-      id: props.id,
-      field: 'account',
-      value: '',
-    });
-  };
-
-  return <GridEditSingleSelectCell onValueChange={handleValueChange} {...props} />;
-};
-```
-
-The demo below combines the steps showed above.
-You can experiment it by changing the value of any cell in the **Type** column.
-The **Account** column is automatically updated with the correct options.
-
-{{"demo": "LinkedFieldsRowEditing.js", "bg": "inline", "defaultCodeOpen": false}}
-
-:::warning
-The call to `apiRef.current.setEditCellValue` returns a promise that must be awaited.
-For instance, if the `singleSelect` column type is used, not awaiting will cause the other column to be rendered with a `value` that is not in the options.
-
-```ts
-const handleChange = async () => {
-  await apiRef.current.setEditCellValue({
-    id: props.id,
-    field: 'account',
-    value: '',
-  });
-};
-```
-
-:::
-
-A similar behavior can be reproduced with cell editing.
-Instead of `apiRef.current.setEditCellValue`, the `rows` prop must be updated or `apiRef.current.updateRows` be used.
-Note that the `onCellEditStart` and `onCellEditStop` props also have to be used to revert the value of the cell changed, in case the user cancels the edit.
-
-{{"demo": "LinkedFieldsCellEditing.js", "bg": "inline", "defaultCodeOpen": false}}
-
-### Usage with `@mui/x-date-pickers`
-
-By default, the grid uses native browser inputs for editing `date` and `dateTime` columns.
-
-While [MUI X Date / Time Pickers](/x/react-date-pickers/getting-started/) are not supported by the grid out of the box yet, it is easy to integrate them by creating [custom edit components](/x/react-data-grid/editing/#create-your-own-edit-component) and [custom filter operators](/x/react-data-grid/filtering/#create-a-custom-operator).
-
-The example below uses `@mui/x-date-pickers` for both `date` and `dateTime` column types:
-
-{{"demo": "EditingWithDatePickers.js", "bg": "inline", "defaultCodeOpen": false }}
-
-:::warning
-You can change date format by importing different locale (`en-US` locale is used in the example above).
-See [Localization](/x/react-date-pickers/localization/) for more information.
-:::
+See [Recipes](/x/react-data-grid/recipes/#editing) for more advanced use cases.
```

Comment:
Could you also create an advanced use cases in https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/row-grouping/ and link to the recipes pages?